src/main/java/org/wecancodeit/pantryplus/controllers/PantryController.java

```diff
@@ -44,6 +44,9 @@ public String userFormProcessing(@RequestParam String firstName, @RequestParam S
 			@RequestParam int familySize, @RequestParam int schoolkidsCount,
 			@RequestParam(defaultValue = "false") boolean infants, @RequestParam String pickUpDate,
 			@RequestParam String zipCode, @RequestParam String birthdate, @RequestParam String address) {
+		if (zipCode.equals("Other")){
+			return "invalid-zipcode";
```

Comment:
`return "invalid-zipcode";` should be a redirect. When I tried playing with this branch and gave an `Other` Zipcode the url I saw in my address bar was: `http://localhost:8080/user-form?firstName=foo&lastName=bar&birthdate=2017-12-31&address=3846+Overdale+Drive&zipCode=Other&familySize=1&infants=true&schoolkidsCount=0&pickUpDate=2018-04-17`

Reply:
Do you have a mapping you would like me to create or would you like to see the page through the Spring framework's default error handler?

Reply:
I would recommend creating a mapping. There's a similar mapping in the EmailController
```Java
@RequestMapping("/email-failure")
public String emailFailure(@RequestParam String error, Model model) {
	model.addAttribute("error", error);
	return "email-failure";
}
```
In this case, we `email-failure.html` is a template, rather than static. We did this because we wanted to pass the stack error through to the front end to make it easier to debug because an email would only fail to send it there was a bug, we thought. Since we know failure to proceed due to the wrong zip code has no associated error there should be no error report. I would also recommend using the same stylesheets that `email-failure.html` uses to save yourself some time.